products/sle15/profiles/pci-dss.profile

```diff
@@ -33,9 +33,6 @@ selections:
     - accounts_password_pam_unix_remember
     - accounts_passwords_pam_faillock_deny
     - accounts_passwords_pam_faillock_unlock_time
-    - accounts_password_pam_ucredit
-    - accounts_password_pam_minlen
-    - accounts_password_pam_dcredit
```

Comment:
This probably shouldn't be here

Reply:
Yes - you are right. I didn't see this before, but it looks like those rules are already above in the file, I think I may have moved them but not sure how to get rid of this?

Reply:
Simply copy-paste the selections from here to where they belong in the actual file, and make another commit. I think that we can squash commits in this PR at the end, as the PR represents only one logical change.

Reply:
Copy-pasted them, and it has the same issue as here just on different lines :)